stable/velero/README.md

```diff
@@ -91,7 +91,7 @@ Parameter | Description | Default
 `metrics.serviceMonitor.enabled` | Set this to `true` to create ServiceMonitor for Prometheus operator | `false`
 `metrics.serviceMonitor.additionalLabels` | Additional labels that can be used so ServiceMonitor will be discovered by Prometheus | `{}`
 `schedules` | A dict of schedules | `{}`
-
+`restic.privileged` | Whether restic should run as a privileged pod. Only necessary in special cases (SELinux) | `false`
```

Comment:
Please, rename this property to `deployRestic.privileged` for consistency

Reply:
This was intentional, as `deployRestic` is currently a boolean and defaults to `false`. Changing it to a map would break existing configs.

Reply:
It'd be good to find another property name to deployRestic 🤔 maybe `restic.deploy` or `restic.enabled`

Reply:
I think `restic.enabled` would be a good name, as most of the Helm charts I know use `enabled` for enabling and disabling components. This would break older deployments though...